test/extensions/filters/http/cache/cache_filter_integration_test.cc

```diff
@@ -108,9 +108,14 @@ class CacheIntegrationTest : public Event::TestUsingSimulatedTime,
   OptRef<const Http::TestResponseTrailerMapImpl> empty_trailers_;
 };
 
-INSTANTIATE_TEST_SUITE_P(Protocols, CacheIntegrationTest,
-                         testing::ValuesIn(HttpProtocolIntegrationTest::getProtocolTestParams()),
-                         HttpProtocolIntegrationTest::protocolTestParamsToString);
+// Test for HTTP1/2 only.
```

Comment:
I'd remove this comment since it's a bit redundant with the TODO and the code below.

https://github.com/envoyproxy/envoy/pull/26107#discussion_r1140667167

(Here and elsewhere)